File: 001-Bit-counting/main.cpp
#include <iostream>
#include <bitset>

using namespace std;

int count_bits(int num) {
    string bit_version = bitset<32>(num).to_string();
    int count = 0;

    for( char x : bit_version) {
        if( x == '1') {
            count++;
        }
    }
    return count;
}

int main () {

    int result = count_bits(1234);

    cout << "The number of bits are: " << result << endl;

    return 0;
}

File: 001-Bit-counting/main.py
def count_bits(n):
    count = 0
    if (n >= 0):
        bit_version = bin(n)[2:]
        
    for x in str(bit_version):
        if x == '1':
            count += 1
    return count

result = count_bits(1234)
print("The number of 1s is: ", result)